packages/contracts/contracts/Router.sol

```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.4;
+
+import "./interfaces/ITransactionManager.sol";
+import "./lib/LibAsset.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+
+contract Router is Ownable {
```

Comment:
should we do `ProposedOwnable` instead?

Reply:
transferring ownership in constructor only, which is going to be two steps process under ProposedOwnable